impeller/display_list/aiks_dl_runtime_effect_unittests.cc

```diff
@@ -83,5 +85,37 @@ TEST_P(AiksTest, DrawPaintTransformsBounds) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+TEST_P(AiksTest, CanRenderRuntimeEffectFilter) {
+  if (GetParam() == PlaygroundBackend::kOpenGLES) {
+    GTEST_SKIP() << "Not currently supported on OpenGLES backend.";
```

Comment:
A note about why?

Reply:
No good reason, just fixed it - wasn't keeping shader metadata alive long enough